src/Bvc.Game.Quest.Tests/Api/GameQuestApiTest.cs

```diff
@@ -48,6 +48,23 @@ public void PostAnotherAchievement()
     {
         //Arrange - create a Player who already has one achievement.
         //Act - call the Api - PostAchievement: same gamerId,  another AchievementId
+        var requestDto = new PostAchievementRequest { GamerId = 0, AchievementId = 0};
+        var requestDto1 = new PostAchievementRequest { GamerId = 0, AchievementId = 2 };
+        
+        var achievementDto = new AchievementDto { Id = requestDto.AchievementId, GamerId = requestDto.GamerId };
+        var achievementDto1 = new AchievementDto { Id = requestDto1.AchievementId, GamerId = requestDto1.GamerId };
+        var returnedFromService = new GamerDto { Id = requestDto.GamerId, Achievement = achievementDto};
```

Comment:
when you mock out the service call.  you need to Setup the service to return a GamerDTO with both Achievements in a list